scripts/ci/travis.sh

```diff
@@ -67,35 +67,24 @@ done
 
 ./cardano-sl-tools.root/bin/cardano-wallet-hs2purs
 
+# Generate daedalus-bridge
 pushd daedalus
   nix-shell --run "npm install && npm run build:prod"
   echo $TRAVIS_BUILD_NUMBER > build-id
   cp ../log-config-prod.yaml .
 popd
 
+# Generate explorer frontend
+EXPLORER_EXECUTABLE=$(pwd)/cardano-sl-explorer-static.root/bin/cardano-explorer-hs2purs ./explorer/frontend/scripts/build.sh
```

Comment:
Nothing wrong with a single line, I'm just used to having one line reserved for a variable I guess. Also, I didn't know that this way you could pass a variable to the script, I thought you always needed to _export_ it first. Nice.